### ISSUE TITLE
Scatterer won't work on SSD, Weirdly?

### ISSUE THREAD

**Issue by kunatics** [LGhassen/Scatterer#222]
After I switched my hard drive to my old one, Surprisingly the scatterer works just fine???

Then I switched to my SSD then it start crashing when booting again.

I use the lastest scatterer version with 1.12.5 KSP

**Comment by LGhassen** [comment]
Add KSP.log file

**Comment by kunatics** [comment]
I downloaded some mod just now, (Mostly visual enhancement mods for RSS)

I booted up the game this time it didn't crash but the scatterer didn't worked (No detailed atmosphere at the game menu)
and when I tried to enter the game saves the game loaded for a second (without detailed water body and atmosphere scatters) then crashes weirdly?

[KSP.log](https://github.com/user-attachments/files/17520836/KSP.log)

**Comment by LGhassen** [comment]
Logs don't actually show any issues, a crash or any problems with scatterer. Maybe your pagefile is set to a small size on the SSD so check that.

**Comment by kunatics** [comment]
Thank you for the review, I will try to fix that.

**Comment by kunatics** [comment]
I increased the paging file size to the recommended, However no luck.

After you log into the game it will run the game for 1 second then after the scatterer supposedly "Loaded in" the game will go dark screen then after that the non-responding and white screen.

I suggest one of the reason why I don't have actual error reports because it was un-responding instead of usual KSP crash.

Here is my KSP.log and error.log

[KSP.log](https://github.com/user-attachments/files/17523364/KSP.log)
[Player.log](https://github.com/user-attachments/files/17523386/Player.log)

**Comment by LGhassen** [comment]
I don't know what to suggest as I can't really see anything going on in the logs.

Usually when the game stops responding on atmosphere generation/loading on lower-end GPUs it's because of this issue https://github.com/LGhassen/Scatterer/issues/168#issuecomment-1938993859 which you can work around by following steps in the last comment to increase the driver timeout, otherwise you can go in the KSC menu, open scatterer settings (blue orb icon) press customize and look for the setting "use low resolution atmospheres", you'll need to delete the folder ScattererAtmosphereCache for both solutions. However in this case not sure this is the same issue at play as you had it working before. Is this a new windows install different from the one you used on the HDD, or is it cloned/moved from the HDD to the SSD?

**Comment by kunatics** [comment]
Yes a new Windows install, Not clone.

I will try all the solution you listen above, Thank you.

**Comment by kunatics** [comment]
I tried increasing the GPU timeout, It didn't work but this time I got a proper crash!

Here are both the Error.log and Player.log hope this atleast clear things up a bit.

[error.log](https://github.com/user-attachments/files/17530161/error.log)
[Player.log](https://github.com/user-attachments/files/17530164/Player.log)

**Comment by LGhassen** [comment]
No idea what's going on at all

**Comment by kunatics** [comment]
Alright thanks, I will close this for now. It seems that the problem is probably with my GPU or CPU I will try upgrading my PC and see.

**Comment by kunatics** [comment]
Okay after a little bit of reading, I found a solution where you use healthy ScattererAtmosphereCache from another person and plug it in your own. Turn out it works!

So if anybody having a crash problem try importing ScattererAtmosphereCache from someone that is able to run it.

**Comment by LGhassen** [comment]
Yeah that shouldn't work, atmosphereCaches aren't transferable between different matchines, no idea how you got this to work.

**Comment by kunatics** [comment]
I have no idea either, I'm suggesting that my GPU isn't able to create a complete atmosphere cache so I use the files from other people to guide my GPU...?

I honestly don't know but it works?

First I clean install using CKAN (Delete everything and reinstall) by installing ONLY scatterer, First time I booted it up it has white rings around the planet. So I asked a guy with working game to send me his AtmosphereCache files and insert it in my own game to kind of act like a guide to my game? Then when I booted it up it worked. Probably some miracle but It worked nontheless. 

My guide to fix the thing would probably be Clean install > Boot the game > Use ScattererAtmosphereCache file from another person
